XRTK-Core/Packages/com.xrtk.core/Interfaces/IMixedRealityService.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) XRTK. All rights reserved.
```

Comment:
Is it okay to change these when there is no substantial change to the code?

Reply:
It's a key differentiator between the code we created for Microsoft and the content generated now as the XRTK. We are very keen on ensuring our licenses reflect our efforts.